test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -549,9 +549,29 @@ private void wipeCluster() throws Exception {
                         adminClient().performRequest(new Request("DELETE", "_template/" + template));
                     }
                 }
+                try {
+                    adminClient().performRequest(new Request("DELETE", "_index_template/*"));
+                    adminClient().performRequest(new Request("DELETE", "_component_template/*"));
```

Comment:
I would have expected that this change was needed in the change that added component templates and not in this change? (not questioning the need for it, but timing)

Reply:
So the index and component templates hung around after YAML tests, but because there was never any warnings/errors, and because they aren't actually used for index creation yet, they didn't cause any errors. This is required now because they were hanging around and then the regular template stuff caused errors/warnings.

Reply:
thanks, makes sense.